RIOT-OS/RIOT
# Kconfig: Expose pkg/wakaama configurations

### Contribution description
This PR moves the LwM2M Wakaama client configuration macros to the `CONFIG_` namespace and exposes them to Kconfig.

### Testing procedure
- The Wakaama example should work as usual
- You should be able to configure the client by running `make menuconfig`. Test e.g. changing the server URI and device name, enabling the package logs, and enabling the usage of a bootstrap server.

### Issues/PRs references
Part of #12888
~~Depends on #12913~~
